smartsim/ml/tf/utils.py

```diff
@@ -83,14 +85,15 @@ def serialize_model(model: keras.Model) -> t.Tuple[str, t.List[str], t.List[str]
 
     to use a Keras or TensorFlow model in SmartSim, the model
     must be frozen and the inputs and outputs provided to the
-    smartredis.client.set_model() method.
+    None.client.set_model() method.
```

Comment:
This whole paragraph also seems very SSv0.7-centric. Feel free to drop!

Reply:
removed